src/content/docs/en/guides/integrations-guide/deno.mdx

```diff
@@ -1,188 +1,13 @@
 ---
-# NOTE: This file is auto-generated from 'scripts/generate-integration-pages.ts'
-#       and pulls content directly from the package’s README.
-#       DO NOT MAKE EDITS TO THIS FILE DIRECTLY, THEY WILL BE OVERWRITTEN!
-#       For corrections, please edit the package README at
-#       https://github.com/withastro/astro/tree/main/packages/integrations/deno/
-#
-# TRANSLATORS: please remove this note and the <DontEditWarning/> component.
-
 type: integration
```

Comment:
We can remove this as proposed by @sarah11918 
```suggestion
```

Reply:
Removed